pkgpanda/util.py

```diff
@@ -150,6 +151,38 @@ def get_requests_retry_session(max_retries=4, backoff_factor=1, status_forcelist
     return session
 
 
+def stream_remote_file_with_retries(out_filename, url, retries=4):
```

Comment:
`retries=4`! Let's explain all of these choices, even if the explanation is "this is my best guess for a compromise and it seems to work". That means that the next maintainer will know whether it is safe to change.

Reply:
This is taken from here:

https://github.com/dcos/dcos/blob/2897e2459e460033fb2b72099d4aac29e2b58937/pkgpanda/util.py#L138

No idea why it's `4` but I thought I'd keep the behaviour the same.